.travis.yml

```diff
@@ -7,7 +9,7 @@ php:
   - 7.0
   - 7.1
   - nightly
-  - hhvm
+  - hhvm-3.6
```

Comment:
Default version of HHVM on Precise is 3.6.6, on Trusty - 3.15.2, see https://github.com/yiisoft/yii2/pull/12971#issuecomment-260134653. That causes failing of 2 tests(#12981, #12982). By specifying exact version we keep the same version as on Precise until these 2 bugs will be fixed. In this case tests are passing without failures.